src/main/scala/io/iohk/ethereum/db/dataSource/EphemDataSource.scala

```diff
@@ -10,6 +10,12 @@ case class EphemDataSource(storage: Map[IndexedSeq[Byte], IndexedSeq[Byte]]) ext
       storage + ((namespace ++ toUpdate._1) -> toUpdate._2))
     EphemDataSource(afterUpdate)
   }
+
+  override def clear: DataSource = EphemDataSource(Map())
+
+  override def close(): Unit = Unit
+
+  override def destroy(): Unit = Unit
```

Comment:
TLDR: it should be 

```
override def destroy(): Unit = ()
```

Explanation: funny thing... it will work just fine, but so would:

```
override def destroy(): Unit = "Unit"
```

Whatever you return will match the `Unit` type. But the sole expression `Unit` is not of type `Unit` but `Unit.type` 😁 A literal for the `Unit` type is `()`